boofuzz/blocks/checksum.py

```diff
@@ -28,6 +28,9 @@ class Checksum(primitives.BasePrimitive):
     the algorithm parameter.
 
     The length field is only necessary for custom algorithms.
+    When using your own provided custom checksum function return is the calculated crc of the data.
```

Comment:
Suggested rewording:
```
When using your own provided custom checksum function, the return value should be the calculated checksum of the data.
```

Reply:
Updated the wording and simplified it slightly to fit the line length limit of 120 characters.